### PR TITLE
[fix](index change) disable light index change for cloud mode

### PR DESCRIPTION
**Pull request by xiaokang**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

File: be/src/olap/schema_change.cpp [hunk 1]
```diff
@@ -1296,6 +1296,17 @@ Status SchemaChangeJob::parse_request(const SchemaChangeParams& sc_params,
         if (column_mapping->expr != nullptr) {
             *sc_directly = true;
             return Status::OK();
+        } else if (column_mapping->ref_column >= 0) {
+            auto column_new = new_tablet_schema->column(i);
```
Comment (platoneko) [inline be/src/olap/schema_change.cpp:1300]:
```suggestion
            const auto& column_new = new_tablet_schema->column(i);
```

File: be/src/olap/schema_change.cpp [hunk 2]
```diff
@@ -1296,6 +1296,17 @@ Status SchemaChangeJob::parse_request(const SchemaChangeParams& sc_params,
...
+            auto column_old = base_tablet_schema->column(column_mapping->ref_column);
```
Comment (platoneko) [inline be/src/olap/schema_change.cpp:1301]:
```suggestion
           const auto& column_old = base_tablet_schema->column(column_mapping->ref_column);
```

**Review by platoneko — CHANGES_REQUESTED** [review]
Please resolve the comment

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Index.java [hunk 3]
```diff
@@ -148,6 +148,14 @@ public Map<String, String> getInvertedIndexCharFilter() {
         return InvertedIndexUtil.getInvertedIndexCharFilter(properties);
     }
 
+    public boolean isLightIndexChangeSupported() {
+        if (indexType == IndexDef.IndexType.INVERTED) {
+            return true;
+        } else {
+            return false;
+        }
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Index.java:156]:
```suggestion
        return indexType == IndexDef.IndexType.INVERTED;
```

**Review by luwei16 — APPROVED** [review]

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
> Please resolve the comment

@platoneko done

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by platoneko — APPROVED** [review]

**Pull request merged**